### PR TITLE
docs: Add HTML Elements Supported to the documentation.

### PR DESCRIPTION
**Pull request by xiCO2k**
This PR adds to the `README.md` the docs about the HTML elements supported.

@nunomaduro can you take a look on this and check if is a good approach for the documentation?

Thanks.

### CODE REVIEW

**Comment by nunomaduro** [comment]
I am not sure we should even document this. Shouldn't we just assume that all elements are supported?

**Comment by xiCO2k** [comment]
You are right, and for us it may be used like a `<div>`.

I see a few cases like `<small>`, `<sub>` and `<sup>` that does not work well for terminal.

May be just do the tailwindcss `classes` ones instead?

File: README.md [hunk 1]
```diff
@@ -75,6 +75,252 @@ style('btn')->apply('p-4 bg-blue text-white');
...
+    </div>
+HTML);
+```
+
+### `<table>`
+
+The `<table>` element can have columns and rows.
+
+```php
+render(<<<'HTML'
```
Comment (butschster) [inline README.md:277]: Look at full example for table element. I think it's important to show all features

```html
<table style="box-double">
    <thead title="Books">
        <tr>
            <th align="right">ISBN</th>
            <th>Title</th>
            <th>Author</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <th align="right">99921-58-10-7</th>
            <td>Divine Comedy</td>
            <td align="right">Dante Alighieri</td>
        </tr>
        <tr border="1">
            <th class="bg-blue text-red" align="right">9971-5-0210-0</th>
            <td>A Tale of Two Cities</td>
            <td align="right">Charles Dickens</td>
        </tr>
        <tr>
            <th align="right">960-425-059-0</th>
            <td>The Lord of the Rings</td>
            <td align="right">J. R. R. Tolkien</td>
        </tr>
        <tr>
            <th align="right">80-902734-1-6</th>
            <td>And Then There Were None</td>
            <td rowspan="2" align="right">Dante Alighieri<br>spans multiple rows</td>
        </tr>
        <tr>
            <th align="right">978-052156781</th>
            <td>De Monarchia</td>
        </tr>
    </tbody>
    <tfoot title="Page 1/2">
        <tr>
            <th colspan="3">This value spans 3 columns.</th>
        </tr>
    </tfoot>
</table>
```

And also tell about table styles
 - default
 - borderless
 - compact
 - box
 - box-double
 - symfony-style-guide

Comment (butschster) [reply README.md:277]: And mention table cells can't have inner elements, only text

Comment (xiCO2k) [reply README.md:277]: Yes the table styles are unique.

**Comment by xiCO2k** [comment]
Hey @butschster do you want to contribute with those `<table>` cases for the documentation?

**Pull request merged**